packages/react-native-layout/package.json

```diff
@@ -7,20 +7,6 @@
   "types": "lib/typescript/index.d.ts",
   "react-native": "src/index",
   "source": "src/index",
-  "files": [
-    "src",
-    "lib",
-    "android",
-    "ios",
-    "cpp",
-    "bothrs-layout.podspec",
-    "!lib/typescript/example",
-    "!android/build",
-    "!ios/build",
-    "!**/__tests__",
-    "!**/__fixtures__",
-    "!**/__mocks__"
-  ],
```

Comment:
Why would you remove this?

Reply:
This is a draft where i was testing a ton of things out in order to make eslint work. Got it figured out now and this'll be fixed in the next commit / actual PR :)